ci-operator/step-registry/tls-13/tls-13-commands.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -euo pipefail
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+oc patch apiservers/cluster --type=merge -p '{"spec": {"tlsSecurityProfile":{"modern":{},"type":"Modern"}}}'
+
+oc adm wait-for-stable-cluster
```

Comment:
ok, i think that the `--minimum-stable-period` is `5m` and `--timeout` is `1h`

Reply:
In my tests, 30mins is enough. 
see https://github.com/openshift/release/pull/63646/files#diff-65cce809dc5ddaa91711f6709acd41e5c24b735c073708e60f10aae28faf8712R851

Reply:
Usually, after applied the apiserver config, we wait a few mins(2-3) for operator's rolling out.

Reply:
I think that our plan will be to slightly [adjust the e2e test](https://github.com/openshift/origin/pull/29611) in origin to read the current profile and perform checks on the affected components. The checks will ensure the components accept only the required TLS profile.